rts/Lua/LuaOpenGL.cpp

```diff
@@ -7050,14 +7052,20 @@ int LuaOpenGL::ObjectLabel(lua_State* L) {
 	return 0;
 }
 
-// https://registry.khronos.org/OpenGL-Refpages/gl4/html/glPushDebugGroup.xhtml
 /***
- * Pushes a debug marker for nVidia nSight 2024.04, does not seem to work when
- * FBO's are raw bound.
- * 
+ * Pushes a debug marker for debugging tools such as `nVidia nSight 2024.04`,
+ * see https://registry.khronos.org/OpenGL-Refpages/gl4/html/glPushDebugGroup.xhtml .
+ *
+ * May be unavailable and `nil` if the platform doesn't support the feature.
```

Comment:
Personally I would've preferred #2022 for these specific functions but this is something that applies to others that can't get the same treatment. Is there some way to handle this boilerplate line systematically?

Reply:
@sprunk I am thinking of adding a system for boilerplate injection to lua-doc-extractor, but I haven't done it yet.

Something like: `@boilerplate messageName` and then have it look in a JSON config file for boilerplate messages. However I haven't yet. Just copy+paste.

Reply:
Not sure if worth it, maybe it's good to keep it simple and do it manually.